test/logger.js

```diff
@@ -67,6 +71,15 @@ describe('Test all Levels', function() {
             done();
         }, configs.FLUSH_INTERVAL + 200);
     });
+    it('Debug Functio, when called with callback, should call the cb', function(done) {
```

Comment:
While it's up to you, I would add a nested `describe` for these (`describe('Debug Function'...) so that the actual test names only say the expected behavior (`it('Executes callback when provided'...)`)